cmd/ingester/app/flags.go

```diff
@@ -54,44 +58,56 @@ const (
 	DefaultParallelism = 1000
 	// DefaultEncoding is the default span encoding
 	DefaultEncoding = EncodingProto
+	// DefaultHTTPPort is the default HTTP port (e.g. for /metrics)
+	DefaultHTTPPort = 14271
+	// IngesterDefaultHealthCheckHTTPPort is the default HTTP Port for health check
+	IngesterDefaultHealthCheckHTTPPort = 14270
```

Comment:
nit: Does this need to have the `Ingester` prefix - as the other constants don't?

Reply:
I've taken the variable names from here and both are prefixed: 

https://github.com/jaegertracing/jaeger/blob/bbf8b7a1eedefa79eefeaef894bbf10928d0a125/cmd/query/app/flags.go#L29
https://github.com/jaegertracing/jaeger/blob/e93c37503532ab7a18a3bdf21eee98194616017f/cmd/collector/app/builder/builder_flags.go#L33

That being said, there's a prefix on the other variable as well